index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <title>tic tac toe</title>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+
+  </head>
+
+  <body onload="begin();">
+    <h1>Let's play tic tac toe!</h1>
+    <div id="message">alerts will appear here</div>
+    <table border="1">
+      <tr>
+        <td class="square" id="squareOne" onclick="nextTurn(this);nextMove(this.id);"></td>
+        <td class="square" id="squareTwo" onclick="nextTurn(this);nextMove(this.id);"></td>
+        <td class="square" id="square=Three" onclick="nextTurn(this);nextMove(this.id);"></td>
```

Comment:
Part of your issue is this mistyped id ("square=Three").  This causes some of the javascript to fail (`nextMove` line 25, 36), which means it exits at the failure and doesn't continue on.   This forces your style changes to only affect the first two cells.  With that id fixed, you will find need to handle the multiple calls to `nextMove`.